sigs.yaml

```diff
@@ -3697,6 +3697,44 @@ workinggroups:
     liaison:
       github: saschagrunert
       name: Sascha Grunert
+- dir: wg-node-lifecycle
+  name: Node Lifecycle
+  mission_statement: >
+    Explore and improve node and pod lifecycle in Kubernetes. This should result in
+    better node drain/maintenance support and better pod disruption/termination. It
+    should also improve node and pod autoscaling, better application migration and
+    availability, load balancing, de/scheduling, node shutdown, cloud provider integrations,
+    and support other new scenarios and integrations.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Apps
+  - Architecture
+  - Autoscaling
+  - CLI
+  - Cloud Provider
+  - Cluster Lifecycle
+  - Node
+  - Scheduling
```

Comment:
Dont we need to add other infra SIGs like storage  as stakeholders too ?

Reply:
+1 added, this list is not final and we will have conversations with each SIG after the KubeCon if they want to be part of this WG

Reply:
I know SIG Storage is impacted as well, I am just not sure how much cooperation is needed.

Reply:
thanks @atiratree 👍 .. I would like to help on this effort anything related to storage.. Tagging leads from storage here.. ( @xing-yang @msau42 @jsafrane @saad-ali )

Reply:
Nice, thanks for the help @humblec! Added.